docs/docs/text/fonts.md

```diff
@@ -9,14 +9,16 @@ In Skia, the `FontMgr` object manages a collection of font families.
 It allows you to access fonts from the system and manage custom fonts.
 
 The `matchFont()` function will always return a font.
+And the font property in `<Text />` is optional.
+
 So a fast way to display text in React Native Skia would be the following:
 
 ```tsx twoslash
 import {Text, matchFont} from "@shopify/react-native-skia";
```

Comment:
we import matchFont but don't use it so it's confusing a bit

maybe we can add a separate Text element in the Demo without matchFont to show that it's optional and will work without it as well?